.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you exclude `.vs` from the pull request?

Reply:
Ok